packages/cohort/src/replicator/statistics/core.rs

```diff
@@ -0,0 +1,38 @@
+#[derive(Debug, Default, Clone)]
+pub struct ReplicatorStatisticsItem {
```

Comment:
Maybe if we represent this concept as enum then processing would be more straightforward, large "match" statement over items which, in that case, do not need to be optional.